src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/HologramGlobalPlaceholder.java

```diff
@@ -0,0 +1,42 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.CustomLog;
+import me.filoghost.holographicdisplays.api.placeholder.GlobalPlaceholder;
+import org.betonquest.betonquest.BetonQuest;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Defines HolographicDisplays global placeholder <code>{bqg:package:variable}</code>.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.